docs/rules/prefer-reflect.md

```diff
@@ -54,12 +54,12 @@ Examples of **correct** code for this rule when used without exceptions:
 ```js
 /*eslint prefer-reflect: "error"*/
 
-Reflect.apply(undefined, args);
-Reflect.apply(null, args);
+Reflect.apply(obj.foo, undefined, args);
```

Comment:
Oops, sorry, I *think* these should all be calling `foo` instead of `obj.foo` ( i.e. `Reflect.apply(foo, undefined, args);`) to match the incorrect examples. Can someone else confirm?